src/include/ceph_fuse.h

```diff
@@ -18,13 +18,15 @@
 #include "acconfig.h"
 #include <fuse.h>
 
+#define CEPH_FUSE_VERSION FUSE_MAKE_VERSION(CEPH_FUSE_VERSION_MAJOR, CEPH_FUSE_VERSION_MINOR)
```

Comment:
if we are sure that CEPH_FUSE_VERSION is more accurate / correct than `FUSE_VERSION`, probably we could just override the latter with the calculated one?

Reply:
If the header order issue is okay above, then I will fix it.